deploy/kubernetes/elastic-agent-managed-kubernetes.yaml

```diff
@@ -28,10 +28,13 @@ spec:
               value: "1"
               # Set to true in case of insecure or unverified HTTP
             - name: FLEET_INSECURE
-              value: false
+              value: "false"
```

Comment:
Maybe setting this as true by default is better as the base case?

Reply:
Ok I don't have a strong argument on that.